woocommerce/woocommerce-android
# HACK Week: AI Sales Price Advisor

Please do not merge until target branch is `trunk`

Adds a sales price advisor on the product pricing screen, if there's no current sales price set. This AI-powered feature suggests a good sales price and when to implement them, based on the store location and current date.

As this is a HACK week work, the design/wording will likely need further iteration. For now please focus on functionality and code for the review.

Please check out the demo below:

[ai-sales-price-advise-demo.webm](https://github.com/woocommerce/woocommerce-android/assets/266376/21b0ed68-4c7c-4feb-9a19-dd37acf6c272)

### Testing instructions


1. Use a WordPress.com site,
2. Go to a Product, then open the product pricing screen,
3. Tap the "Recommend me a sales price" button,
4. Wait and ensure the advice is shown.
